package.json

```diff
@@ -75,6 +75,15 @@
 					"light": "images/light/Open_Project.svg"
 				}
 			},
+			{
+				"command": "autolisp.closeProject",
+				"category": "AutoLISP",
+				"title": "%autolispext.project.closeproject.title%",
+				"icon": {
+					"dark": "images/dark/close.svg",
```

Comment:
I am not sure if we need a professional designer to give us two icons, or Microsoft has some standard icons.

Reply:
@Sen-real do you have some comments on these icons?

Reply:
@hualin-adsk I'm the one who uploaded these two icons